operator/apis/machinelearning.seldon.io/v1/seldondeployment_types.go

```diff
@@ -602,6 +603,7 @@ type PredictiveUnit struct {
 	EnvSecretRefName        string                        `json:"envSecretRefName,omitempty" protobuf:"bytes,10,opt,name=envSecretRefName"`
 	StorageInitializerImage string                        `json:"storageInitializerImage,omitempty" protobuf:"bytes,11,opt,name=storageInitializerImage"`
 	Logger                  *Logger                       `json:"logger,omitempty" protobuf:"bytes,12,opt,name=logger"`
+	RunAsUid                *int64                        `json:"runAsUid,omitempty" protobuf:"bytes,13,opt,name=runAsUid"`
```

Comment:
why can't a user ad runAsUid to custom PodSpecs for their component?

Reply:
@cliveseldon Hi~ thanks for your review.
Network problem with initcontainer after using Istio is a common problem. We have encountered it since OKD 3.11 and there are many issues in the Istio community discussing this problem. I think that if it is a common problem and is caused by the seldon-core deployment method, then it would be a better approach to have it solved by the seldon-operator itself.
Also, my personal opinion is that using a custom initcontainer would solve the problem, but it would be very difficult for many data scientists who are more focused on the business or who are not very proficient in the underlying k8s/seldon to write a more complex resource Yaml.